components/brave_vpn/browser/connection/ikev2/win/brave_vpn_ras_connection_api_win.cc

```diff
@@ -39,6 +39,30 @@ RasOperationResult DisconnectEntry(const std::wstring& name) {
   return brave_vpn::ras::DisconnectEntry(name);
 }
 
+bool IsInternetAvailable() {
```

Comment:
nit: maybe `IsInternetAvailable()` is removed as `IsPlatformNetworkAvailable` only uses this?